<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Week 2 Spot Assays and Purification | 2021 Third Year Phage Hunters</title>
  <meta name="description" content="A handbook." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="Week 2 Spot Assays and Purification | 2021 Third Year Phage Hunters" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="/images/logo.v3.png" />
  <meta property="og:description" content="A handbook." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Week 2 Spot Assays and Purification | 2021 Third Year Phage Hunters" />
  
  <meta name="twitter:description" content="A handbook." />
  <meta name="twitter:image" content="/images/logo.v3.png" />




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="phage-enrichment.html"/>
<link rel="next" href="phage-purification-and-dna-extraction.html"/>
<script src="libs/header-attrs-2.7/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="./">Citizen Phage Library</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome Phage Hunters!</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#project-aims"><i class="fa fa-check"></i>Project Aims</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#expected-learning-outcomes"><i class="fa fa-check"></i>Expected Learning Outcomes</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#your-supervisory-team-for-this-project"><i class="fa fa-check"></i>Your Supervisory Team for this project</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="house-keeping-rules.html"><a href="house-keeping-rules.html"><i class="fa fa-check"></i>House Keeping Rules</a>
<ul>
<li class="chapter" data-level="" data-path="house-keeping-rules.html"><a href="house-keeping-rules.html#lab-times-and-kick-off-meeting"><i class="fa fa-check"></i>Lab times and Kick-Off Meeting</a></li>
<li class="chapter" data-level="" data-path="house-keeping-rules.html"><a href="house-keeping-rules.html#safe-working-practices"><i class="fa fa-check"></i>Safe working practices</a></li>
<li class="chapter" data-level="" data-path="house-keeping-rules.html"><a href="house-keeping-rules.html#working-in-a-covid-secure-manner"><i class="fa fa-check"></i>Working in a COVID-secure manner</a></li>
<li class="chapter" data-level="" data-path="house-keeping-rules.html"><a href="house-keeping-rules.html#aseptic-technique"><i class="fa fa-check"></i>Aseptic technique</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="phage-enrichment.html"><a href="phage-enrichment.html"><i class="fa fa-check"></i><b>1</b> Phage Enrichment</a>
<ul>
<li class="chapter" data-level="1.1" data-path="phage-enrichment.html"><a href="phage-enrichment.html#monday-17th-may"><i class="fa fa-check"></i><b>1.1</b> Monday 17th May</a></li>
<li class="chapter" data-level="1.2" data-path="phage-enrichment.html"><a href="phage-enrichment.html#tuesday-18th-may"><i class="fa fa-check"></i><b>1.2</b> Tuesday 18th May</a></li>
<li class="chapter" data-level="1.3" data-path="phage-enrichment.html"><a href="phage-enrichment.html#wednedsay-19th-may"><i class="fa fa-check"></i><b>1.3</b> Wednedsay 19th May</a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="phage-enrichment.html"><a href="phage-enrichment.html#sewage-sample-from-countess-wear"><i class="fa fa-check"></i><b>1.3.1</b> Sewage sample from Countess Wear</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="phage-enrichment.html"><a href="phage-enrichment.html#thursday-20th-may"><i class="fa fa-check"></i><b>1.4</b> Thursday 20th May</a></li>
<li class="chapter" data-level="1.5" data-path="phage-enrichment.html"><a href="phage-enrichment.html#friday-21st-may"><i class="fa fa-check"></i><b>1.5</b> Friday 21st May</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="spot-assays-and-purification.html"><a href="spot-assays-and-purification.html"><i class="fa fa-check"></i><b>2</b> Spot Assays and Purification</a>
<ul>
<li class="chapter" data-level="2.1" data-path="spot-assays-and-purification.html"><a href="spot-assays-and-purification.html#monday-24th-may"><i class="fa fa-check"></i><b>2.1</b> Monday 24th May</a></li>
<li class="chapter" data-level="2.2" data-path="spot-assays-and-purification.html"><a href="spot-assays-and-purification.html#tuesday-25th-may"><i class="fa fa-check"></i><b>2.2</b> Tuesday 25th May</a></li>
<li class="chapter" data-level="2.3" data-path="spot-assays-and-purification.html"><a href="spot-assays-and-purification.html#wednesday-26th-may"><i class="fa fa-check"></i><b>2.3</b> Wednesday 26th May</a></li>
<li class="chapter" data-level="2.4" data-path="spot-assays-and-purification.html"><a href="spot-assays-and-purification.html#thursday-27th-may"><i class="fa fa-check"></i><b>2.4</b> Thursday 27th May</a></li>
<li class="chapter" data-level="2.5" data-path="spot-assays-and-purification.html"><a href="spot-assays-and-purification.html#friday-28th-may"><i class="fa fa-check"></i><b>2.5</b> Friday 28th May</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="phage-purification-and-dna-extraction.html"><a href="phage-purification-and-dna-extraction.html"><i class="fa fa-check"></i><b>3</b> Phage Purification and DNA extraction</a>
<ul>
<li class="chapter" data-level="3.1" data-path="phage-purification-and-dna-extraction.html"><a href="phage-purification-and-dna-extraction.html#tuesday-1st-june"><i class="fa fa-check"></i><b>3.1</b> Tuesday 1st June</a></li>
<li class="chapter" data-level="3.2" data-path="phage-purification-and-dna-extraction.html"><a href="phage-purification-and-dna-extraction.html#wednesday-2nd-june"><i class="fa fa-check"></i><b>3.2</b> Wednesday 2nd June</a></li>
<li class="chapter" data-level="3.3" data-path="phage-purification-and-dna-extraction.html"><a href="phage-purification-and-dna-extraction.html#thursday-3rd-june"><i class="fa fa-check"></i><b>3.3</b> Thursday 3rd June</a></li>
<li class="chapter" data-level="3.4" data-path="phage-purification-and-dna-extraction.html"><a href="phage-purification-and-dna-extraction.html#friday-4th-june"><i class="fa fa-check"></i><b>3.4</b> Friday 4th June</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="genome-annotation.html"><a href="genome-annotation.html"><i class="fa fa-check"></i><b>4</b> Genome Annotation</a>
<ul>
<li class="chapter" data-level="4.1" data-path="genome-annotation.html"><a href="genome-annotation.html#monday-7th-june"><i class="fa fa-check"></i><b>4.1</b> Monday 7th June</a></li>
<li class="chapter" data-level="4.2" data-path="genome-annotation.html"><a href="genome-annotation.html#tuesdsay-8th---friday-11th-june"><i class="fa fa-check"></i><b>4.2</b> Tuesdsay 8th - Friday 11th June</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="recipes-for-reagents.html"><a href="recipes-for-reagents.html"><i class="fa fa-check"></i>Recipes for reagents</a>
<ul>
<li class="chapter" data-level="4.3" data-path="recipes-for-reagents.html"><a href="recipes-for-reagents.html#agar-ploates"><i class="fa fa-check"></i><b>4.3</b> Agar ploates</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="recipes-for-reagents.html"><a href="recipes-for-reagents.html#supplemental-nutrients"><i class="fa fa-check"></i><b>4.3.1</b> Supplemental nutrients</a></li>
<li class="chapter" data-level="4.3.2" data-path="recipes-for-reagents.html"><a href="recipes-for-reagents.html#to-make-800-ml-of-bottom-agar-plates"><i class="fa fa-check"></i><b>4.3.2</b> To make 800 mL of bottom agar plates:</a></li>
<li class="chapter" data-level="4.3.3" data-path="recipes-for-reagents.html"><a href="recipes-for-reagents.html#to-make-800-ml-of-top-agar"><i class="fa fa-check"></i><b>4.3.3</b> To make 800 mL of top agar:</a></li>
<li class="chapter" data-level="4.3.4" data-path="recipes-for-reagents.html"><a href="recipes-for-reagents.html#a-note-on-agar-density-thickness"><i class="fa fa-check"></i><b>4.3.4</b> A note on agar density thickness</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="recipes-for-reagents.html"><a href="recipes-for-reagents.html#precipitant-solution"><i class="fa fa-check"></i><b>4.4</b> Precipitant solution</a></li>
<li class="chapter" data-level="4.5" data-path="recipes-for-reagents.html"><a href="recipes-for-reagents.html#nuclease-solution"><i class="fa fa-check"></i><b>4.5</b> Nuclease solution</a></li>
<li class="chapter" data-level="4.6" data-path="recipes-for-reagents.html"><a href="recipes-for-reagents.html#beef-resuspension-solution"><i class="fa fa-check"></i><b>4.6</b> Beef resuspension solution</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="protocols.html"><a href="protocols.html"><i class="fa fa-check"></i>Protocols</a>
<ul>
<li class="chapter" data-level="4.7" data-path="protocols.html"><a href="protocols.html#preparing-phages-from-environmental-samples"><i class="fa fa-check"></i><b>4.7</b> Preparing phages from environmental samples</a>
<ul>
<li class="chapter" data-level="4.7.1" data-path="protocols.html"><a href="protocols.html#solid-samples"><i class="fa fa-check"></i><b>4.7.1</b> Solid Samples</a></li>
<li class="chapter" data-level="4.7.2" data-path="protocols.html"><a href="protocols.html#liquid-samples"><i class="fa fa-check"></i><b>4.7.2</b> Liquid Samples</a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="protocols.html"><a href="protocols.html#initial-phage-enrichment-using-2.2-ml-deep-well-plates-or-7-ml-sterilins"><i class="fa fa-check"></i><b>4.8</b> 1. Initial phage enrichment (using 2.2 mL deep well plates or 7 mL sterilins)</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="risk-assessment.html"><a href="risk-assessment.html"><i class="fa fa-check"></i><b>5</b> Risk Assessments</a></li>
<li class="chapter" data-level="" data-path="host-strains.html"><a href="host-strains.html"><i class="fa fa-check"></i>Host strains</a></li>
<li class="chapter" data-level="" data-path="outputs.html"><a href="outputs.html"><i class="fa fa-check"></i>Outputs</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">2021 Third Year Phage Hunters</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="spot-assays-and-purification" class="section level1" number="2">
<h1><span class="header-section-number">Week 2</span> Spot Assays and Purification</h1>
<p>Last week, you prepared samples by enriching any phages that might be present for your pathogen. This week, we get to find out if your samples contained any suitable phages. We do this using something called a <em>spot assay</em>. Briefly, we add 5 µL of enriched viral lysate to a lawn of host culture. If there are suitable viruses present in the lysate, they will infect the cells in the lawn and start to replicate. Each round of replication releases more viruses, and a plaque begins to form, spreading outwards in a circular fashion from the initial infection. If you’re interested in how plaques form, I recommend reading the paper by <span class="citation">(Abedon and Yin <a href="#ref-Abedon2009-zt" role="doc-biblioref">2009</a>)</span>.</p>
<div id="monday-24th-may" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Monday 24th May</h2>
<p>Today, we set up the first spot assay to see if we have any viruses in your samples. To make the lawn of bacteria, you need to combine an exponentially growing host with soft agar, then pour it onto a bottom plate and swirl it round for even coverage. You need to work quickly. The molten agar is above the thermal tolerance of most host bacteria, so we need to minimise the time the host spends in hot agar. Once on the bottom plate, the hot agar sets quickly, so you need to swirl quickly to get even coverage before it solidifies.</p>
<p>We’ve set up three pouring stations with water baths containing glass test tubes with 3 mL molten top agar in them. Students will come up in turn and pour their host lawns. You’ll have one extra plate for a practice pour.</p>
<p><img src="images/first-plaque-assay.png" /></p>
<ul class="task-list">
<li><p><input type="checkbox" disabled="" />
<strong>Step 1</strong>: Make a fresh overnight culture for tomorrow’s plaque assays. Flame an inoculation loop until it glows red and then allow it to cool. Pick 3-4 colonies from your overnight streak plate by dabbing the loop on them. Transfer the loop into fresh LB medium in a sterilin. Reflame your loop. Label the sterilin with your name, date and host name. Bring the new streak plate and overnight culture to the front for us to incubate.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 2</strong>: Collect your plate of phage lysates from last week and a culture of your host at <span class="math inline">\(OD_{600}\)</span> of 0.6. You will also need 4 bottom agar plates. You’ll also need to collect a microcentrifuge tube of the phage lysate for your host from the chicken sample (<strong>C</strong>), and a microcentrifuge tube of the phage lysate for your host from the sewage sample (<strong>S</strong>).</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 3</strong>: You’ll be putting 12 spots per plate (one per sample). I find the easiest way to do this is to divide your plate into quadrants, and then place three spots per quadrant. Draw two perpendicular bisecting lines on the bottom of each plate with a ruler to divide it into quadrants. Mark one edge with an arrow so we can orientate it. Label each plate with your name, the host bacterium, the date, and <em>Plate 1</em> through <em>Plate 3</em>. Label the last plate <em>Practice</em>. On your three numbered plates, write the numbers of the samples where you are going to put them in the quadrants as shown here:</p></li>
</ul>
<p><img src="images/plate-layout.png" />
where <strong>B</strong> is where you’ll put your blank, <strong>C</strong> is where you’ll put your chicken sample and <strong>S</strong> is where you’ll put your sewage sample.</p>
<ul class="task-list">
<li><input type="checkbox" disabled="" />
<strong>Step 4a</strong>: Bring your labelled plates and host culture to a pouring station. We’ll walk you through the process. Briefly vortex your host culture (1-2 seconds) to resuspend it. Under a blue flame, put your practice plate in front of you and take off the lid. Take the cap off of a tube of molten agar, but leave it in the water bath. Use a pipette to suck up 1 mL of host culture into a blue tip. Using the other hand, take the tube of molten agar out of the bath and add the 1 mL of culture to it. Immediately vortex the agar (1-2 seconds) to mix and pour quickly but smoothly into the centre of the practice plate. If there are any bubbles in the tube, they’ll come out last, and you want to leave them in the tube. Quickly place the tube into the waste beaker and swirl the plate to ensure even coverage of the agar across the surface. Put the lid back on the plate, but leave it with the agar on the bottom to allow the top agar to set. Slide the plate over to one side and repeat with your remaining three plates. Note the time or set a timer for 15 minutes.</li>
<li><input type="checkbox" disabled="" />
<strong>Step 4b</strong>: After 15 minutes, collect your plates (the top agar should now be set) and take them back to your work area. Spot assays work best if the phages are added shortly after the top agar has set.</li>
<li><input type="checkbox" disabled="" />
<strong>Step 5a</strong>: Under a blue flame, open up Plate 1 and spot 5 µL of sample from the first eleven samples of your 96 well plate, plus your blank onto their appropriate locations on the plate. To do this, gently push the liquid out of the pipette over the location, when it has formed a droplet, touch the droplet to the surface of the agar to transfer.</li>
<li><input type="checkbox" disabled="" />
<strong>Step 5b</strong>: Cover the plate and gently slide it to one side so as not to disturb the drops (or you get long, streaky plaques). As long as it is still near the blue flame, slide the lids partially to one side to speed up drying. <strong>DO NOT INVERT THE PLATE</strong>. It can take 20-30 mins outside of a flowhood for a droplet to dry, so we want to leave it as long as possible. Once the droplet is dry, any phages will diffuse through the agar and find their prey.</li>
<li><input type="checkbox" disabled="" />
<strong>Step 5c</strong>: Repeat the process with the remaining two plates and the remaining samples. As before, if you accidentally put a sample on the wrong spot, make a note of it so we can correct it later.</li>
<li><input type="checkbox" disabled="" />
<strong>Step 5d</strong>: When the last plate is dry, slide the lids back on and invert the plates. Bring the plates to the front and we will incubate them at 37 °C overnight to allow plaque formation. Put a PCR film on the 96-well plate of lysate plate and return it to the front (we’ll keep it just in case!)</li>
</ul>
</div>
<div id="tuesday-25th-may" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Tuesday 25th May</h2>
<p>It’s plaque-picking day! Hopefully, at least one of your samples will have yielded a plaque on one of the hosts we have attempted. With a bit of luck, you are now the discoverer of a new virus (or viruses!). At this stage, there is the possibility that each plaque comprises a mixture of phages.</p>
<p>If phages for the host are common in the sample, (e.g. <em>E. coli</em> phages in sewage), it’s quite likely that more than one suitable phage was in the original overnight inoculum. This is where infection efficiency comes into play in enrichment cultures - phages that replicate faster than other phages will produce more progeny. Following the first round of enrichment, the fastest phages will be the most abundant. The second round of enrichment in pure culture provides a second round of purifying selection. So, even if the initial sample had lots of suitable phages, it’s reasonably likely that by this stage, your lysate contains only one or two phages.</p>
<p>So, if we are interested in recovering the maximum <em>diversity</em> of phages from a sample, then enrichment is not advised, if samples contain lots of different phages. To maximise diversity, the best way is to just do directly spot the original sample on a lawn of bacteria and look for different plaque morphologies.</p>
<p>For our purposes, we use enrichment for two reasons: First, there are many hosts for which viruses are not common, so we need to amplify them up in order to isolate them. Second, even if a sample contains lots of suitable phages, the ones best suited for phage therapy are those that are the fastest replicating, most virulent phages for that host - i.e. the ones that would be victorious in purifying selection during enrichment.</p>
<p>There is still the risk though that we have multiple fast replicating phages in our lysate. If we were to extract DNA from these now and then sequence them, we run the risk of combining the genomes of two phages. Even if we can assemble them into individual phage genomes, we could end up with a mixture of phages where one is suitable for therapeutic use and the other isn’t. So, we need to purify them into an axenic culture.</p>
<p>We do this through something called ‘Dilution to Extinction’ purification. Briefly, we take a plaque sample, then use it to make a dilution series. We plate out each dilution in a spot assay until we can see individual plaques.</p>
<p><img src="images/spot-dilution.png" /></p>
<p>We then pick a plaque that is from the greatest amount of dilution (lowest concentration of phages). It is possible that there could still be more than one phage in this plaque, if we were unlucky, so we repeat the process another two times. For the rest of this week, we will be performing these purification steps.</p>
<ul>
<li><p><input type="checkbox" disabled="" />
<strong>Step 1</strong>: Make a fresh streak plate of your assigned culture. Flame an inoculation loop until it glows red and then allow it to cool. Pick 3-4 colonies from your overnight streak plate by dabbing the loop on them. Streak them down one side of a petri dish containing LB + agar. Reflame your loop, allow to cool and drag one side perpendicular to the original streak. Turn your plate 90° and repeat. Label with your name, date and host name.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 2</strong>: Make a fresh overnight culture for tomorrow’s plaque assays. Flame an inoculation loop until it glows red and then allow it to cool. Pick 3-4 colonies from your overnight streak plate by dabbing the loop on them. Transfer the loop into fresh LB medium in a sterilin. Reflame your loop. Label the sterilin with your name, date and host name. Bring the new streak plate and overnight culture to the front for us to incubate.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 3</strong>: Collect your plates from the front and note the samples for which you have plaques. Make a record of the features of the plaque (We’ll take photos of each plate with plaques).</p></li>
<li><p><em>How big is it?</em> Plaque size is a function of how easily the virus diffuses through the medium (e.g. size) and how fast it replicates - plaques stop growing when the hosts reach stationary phase in the medium. The faster it replicates, the greater the number of replication cycles, the larger the plaque.</p></li>
<li><p><em>Does it have a halo?</em> A halo is a good indicator of a virulent phage with an explosive escape from the cell powered by lysins. Lysins continue to diffuse through the medium and kill hosts after the viral plaque has stopped growing, creating a halo.</p></li>
<li><p><em>Is the centre clear or cloudy?</em> Cloudy plaques are usually a good indicator of a temperate phage. Temperate phages confer <strong>superinfection immunity</strong> to the cell, preventing infection from other similar phages. Protected from lytic infection in this way, the cells can continue to replicate in the plaque, causing it to be cloudy. It doesn’t reach the same density as the surrounding lawn because the lytic/lysogenic switch is a stochastic process, so some killing continues.</p></li>
</ul>
<p>You have been assigned three phages from your samples on the spreadsheet <a href="https://docs.google.com/spreadsheets/d/1V6doztAX4AQ5657eH5r0GbJj_GnfMxTe4eKh7metXl0/edit?usp=sharing">here</a></p>
<ul class="task-list">
<li><input type="checkbox" disabled="" />
<strong>Step 4a</strong>: Label a microcentrifuge tube containing 100 µL of SM buffer with the phage identifier (e.g. P1), the host and your name.</li>
<li><input type="checkbox" disabled="" />
<strong>Step 4b</strong>: Under a blue flame, take the lid off of your plate. Set a 1000 µL pipette to 200 µL. Using a blue pipette tip take a core of the plaque (we’ll show you how to do this) and eject the agar core into the 100 µL of SM buffer. Vortex the microcentrifuge tube for 2-3 seconds, leave for 5 minutes then vortex again for 2-3 seconds. The viruses will be released from the agar into the SM buffer.</li>
</ul>
<p><img src="images/dte-plate.png" /></p>
<ul class="task-list">
<li><p><input type="checkbox" disabled="" />
<strong>Step 5a</strong>: Into a 96 well plate, set up a dilution series in a row for each plaque you have picked. Pipette <strong>90 µL</strong> of SM buffer into wells 2 through 11 (ignore columns 1 and 12). You can use the same pipette tip for each row.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 5b</strong>: Pipette <strong>100 µL</strong> from your microcentrifuge tube into column 1.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 5c</strong>: Pipette <strong>10 µL</strong> from column 1 into column 2 and mix with gentle pipetting.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 5d</strong>: Pipette <strong>10 µL</strong> from column 2 into column 3 and mix with gentle pipetting.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 5e</strong>: Pipette <strong>10 µL</strong> from column 3 into column 4 and mix with gentle pipetting.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 5f</strong>: Repeat this process of adding <strong>10 µL</strong> of mixture from column <span class="math inline">\(n\)</span> into <span class="math inline">\(n+1\)</span> and mixing until you’ve added 10 µL to column 11 and mixed.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 5g</strong>: Change your pipette tip and do this each microcentrifuge tube you have, with each going into column 1 on a different row.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 6a</strong>: Collect a bottom agar plate for every row you have filled on your 96-well plate.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 6b</strong>: Divide it into quadrants and mark the locations of the dilution series as shown in the figure below.</p></li>
</ul>
<p><img src="images/dte-layout.png" /></p>
<p>The numbers correspond to the columns of the plate. To convert to dilution factor, we subtract 1 and raise it to the power of 10, e.g. Position 1 is undiluted (<span class="math inline">\(10^{0}\)</span>), Position 2 is a 10-fold dilution (<span class="math inline">\(10^{1}\)</span>) etc. Label the plate with your name, the host name, the date and the sample number from which the plaque was picked.</p>
<ul class="task-list">
<li><input type="checkbox" disabled="" />
<strong>Step 7</strong>: Using the same protocol as yesterday, visit a pouring station and make a lawn of bacteria on your plates and allow to set for 15 mins before taking them back to your bench.</li>
<li><input type="checkbox" disabled="" />
<strong>Step 8</strong>: Using the same method as yesterday, spot <strong>5 µL</strong> from each well in a row of your 96-well plate onto their assigned position on the plate. Onto the blank (<strong>B</strong>), pipette <strong>5 µL</strong> of SM buffer. We want to make sure any ‘plaque’ is a result of viral infection, not an interaction with the buffer. Allow the spots to dry before inverting your plate and bringing it to the front for us to put in an incubator overnight.</li>
</ul>
</div>
<div id="wednesday-26th-may" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Wednesday 26th May</h2>
<p>Hopefully, following overnight culture, your plates will look something like this:</p>
<p><img src="images/spot-dilution.png" /></p>
<p>The next step is to core an individual plaque from the largest dilution number (e.g at <span class="math inline">\(10^{-9}\)</span> here) and repeat the dilution to extinction process.</p>
<ul class="task-list">
<li><p><input type="checkbox" disabled="" />
<strong>Step 1</strong>: Make a fresh overnight culture for tomorrow’s plaque assays. Flame an inoculation loop until it glows red and then allow it to cool. Pick 3-4 colonies from your overnight streak plate by dabbing the loop on them. Transfer the loop into fresh LB medium in a sterilin. Reflame your loop. Label the sterilin with your name, date and host name. Bring the new streak plate and overnight culture to the front for us to incubate.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 2</strong>: Label a microcentrifuge tube containing <strong>100 µL</strong> of SM buffer with the number of the sample, the host and your name for each plate from yesterday.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 3</strong>: Under a blue flame, take the lid off of your plate. Take a core of an individual plaque from the largest dilution number. <em>If you have clearly distinct plaque morphologies at high dilutions, it could be an opportunity to isolate another phage, so let us know!</em></p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 4</strong>: Add the core to the 100 µL of SM buffer, vortex, let sit and vortex again as you did yesterday.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 5</strong>: Set up your dilution series on a 96-well plate as you did yesterday.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 6</strong>: Collect your bottom agar plates and label them up as you did yesterday (date, sample, name, host, plus dilution well in quadrants).</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 7</strong>: Prepare your bacterial lawns at a pouring station.</p></li>
<li><p><input type="checkbox" disabled="" />
<strong>Step 8</strong>: Under a blue flame, spot <strong>5 µL</strong> of each dilution onto the freshly set lawn in the appropriate position and allow to dry.</p></li>
</ul>
</div>
<div id="thursday-27th-may" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> Thursday 27th May</h2>
<p>Final Round! Today we repeat what we did yesterday for the final round of purification. At the end of this, plaques are virtually certain to be axenic.</p>
<p><strong>Volunteers Required if you are using <em>E. coli</em> as a host</strong>: As a positive control for our sequencing, we are going to grow a culture of Escherichia phage T4. Because the genome of this is well known, we will be able to evaluate how well we are recovering the genome and the error rate of the reads pre- and post error correction. We can then assume that the same degree of error (should be very small) is found in our novel phage genomes. If you can fit it onto one of your dilution plates, create an extra spot called ‘T4’ and spot <strong>5 µL</strong> of the stock culture we have onto that plate. Because it’s already pure, we don’t need to go through multiple rounds of purification.</p>
</div>
<div id="friday-28th-may" class="section level2" number="2.5">
<h2><span class="header-section-number">2.5</span> Friday 28th May</h2>
<p>We should now have plaques that are single axenic viral cultures. Now we need to bulk them up for sequencing and imaging and for making viral stocks for future use. We will do this by picking a plaque and adding it to an exponential culture of host. In total, we need about 40 mL of host per phage (30 mL for DNA extraction, 500 µL for imaging and the remainder for putting into long term phage storage as part of the Citizen Phage Library). You can’t grow 40 mL of culture in a single Falcon tube - it doesn’t have enough headspace for gas exchange and there’s too much liquid for efficient mixing. So, we’re going to grow it up as two 20 mL cultures and then combine them.</p>
<p>Repeat the following for each phage:</p>
<p><img src="images/bulking-up.png" /></p>
<ul class="task-list">
<li><input type="checkbox" disabled="" />
<strong>Step 1</strong>: Under a blue flame, core a plaque from your highest dilution just like before and place it in a <strong>100 µL</strong> of SM buffer in a microcentrifuge tube. Label the tube with the host and the sample number.</li>
<li><input type="checkbox" disabled="" />
<strong>Step 2</strong>: Vortex briefly and leave it for five minutes before vortexing again.</li>
<li><input type="checkbox" disabled="" />
<strong>Step 3a</strong>: Add <strong>50 µL</strong> of the SM buffer with your core to a Falcon tube containing <strong>20 mL</strong> of LB + 10 mM <span class="math inline">\(MgCl_{2}\)</span> + 10 mM <span class="math inline">\(CaCl_{2}\)</span>.</li>
<li><input type="checkbox" disabled="" />
<strong>Step 3b</strong>: Add a second aliquot of <strong>50 µL</strong> of the SM buffer with your core to another Falcon tube containing <strong>20 mL</strong> of LB + 10 mM <span class="math inline">\(MgCl_{2}\)</span> + 10 mM <span class="math inline">\(CaCl_{2}\)</span>.</li>
<li><input type="checkbox" disabled="" />
<strong>Step 3c</strong>: Label each Falcon tube with your name, the host, the date and the sample number the phage came from.</li>
<li><input type="checkbox" disabled="" />
<strong>Step 4</strong>: Into each Falcon tube, add <strong>500 µL</strong> of overnight host culture. Vortex briefly and return the tubes to the front so we can put them in the incubator. Each student should have <strong>six</strong> tubes (two per phage).</li>
<li><input type="checkbox" disabled="" />
<strong>Step 5</strong>: Return your plaques to the front - we’ll keep them just in case we need to take new plaques.</li>
</ul>
<p>We’ll put your tubes into an incubator at 37 °C overnight and come in tomorrow to transfer them to the fridge for Tuesday. On Tuesday morning, we will centrifuge them at 10,000 <span class="math inline">\(\times g\)</span> for 30 minutes at 4 °C to pellet the cell debris and leave the viruses in the supernatant.</p>
<p>We’ll also create one 20 mL culture of each host with no phages so we can monitor differences between infected and uninfected cultures as a check of whether an active infection is occuring (lower <span class="math inline">\(OD_{600}\)</span>, more debris at the bottom of the Falcon tube, colour change in cultures).</p>
<p>Finally, we’ll pick a good T4 plaque and inoculate two 20 mL cultures of <em>E. coli</em> for our positive control.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references hanging-indent">
<div id="ref-Abedon2009-zt">
<p>Abedon, Stephen T, and John Yin. 2009. “Bacteriophage Plaques: Theory and Analysis.” Edited by Martha R J Clokie and Andrew M Kropinski, 161–74.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="phage-enrichment.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="phage-purification-and-dna-extraction.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["third-year-project.pdf", "third-year-project.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
